packages/playground/src/components/node_resources_charts.vue

```diff
@@ -19,15 +19,14 @@
       <div v-for="item in resources" :key="item.name" class="mx-6 d-flex flex-column pt-2 mt-2 align-center">
         <div class="mb-2">{{ item.name }}</div>
         <div class="text-center">
-          <v-progress-circular :model-value="item.value"
-            >{{ item.value !== "NaN" ? item.value + "%" : "N/A" }}
-          </v-progress-circular>
+          <tf-loading :model-value="item.value"> {{ item.value !== "NaN" ? item.value + "%" : "N/A" }}</tf-loading>
         </div>
       </div>
     </v-row>
 
     <v-row justify="center">
-      <v-progress-circular v-if="loading" indeterminate color="primary" :size="50" class="mt-10 mb-10" />
+      <tf-loading v-if="loading" class="mt-10 mb-10" />
+
```

Comment:
Shouldn't we consider the size as well? I mean if the previous loading size was 50, then this also should be the same. This goes for all the loaders as well. Maybe we can create a size prop with a default size?

Reply:
it's already default 50, if you check **packages/playground/src/plugins/vuetify.ts** you'll find the default props